packages/create-cloudflare/turbo.json

```diff
@@ -3,18 +3,20 @@
 	"extends": ["//"],
 	"tasks": {
 		"build": {
+			"passThroughEnv": ["CI"],
 			"env": [
 				"TEST_PM",
 				"TEST_PM_VERSION",
 				"npm_config_user_agent",
-				"CI",
 				"CREATE_CLOUDFLARE_TELEMETRY_DISABLED",
 				"CREATE_CLOUDFLARE_TELEMETRY_DEBUG",
 				"SPARROW_SOURCE_KEY"
 			],
-			"inputs": ["src/**", "templates*/**", "scripts/**", "*", "vitest*"],
```

Comment:
This includes most folders, so instead just use the turbo default of all git-tracked files